config/initializers/devise.rb

```diff
@@ -186,9 +186,7 @@
   config.email_regexp = /\A[^@\s]+@[^@\s]+\z/
 
   # ==> Configuration for :timeoutable
-  # The time you want to timeout the user session without activity. After this
-  # time the user will be asked for credentials again. Default is 30 minutes.
-  # config.timeout_in = 30.minutes
+  config.timeout_in = 60.minutes
```

Comment:
I doubled the timeout, looks better to me . But it's reversible, @danielmbrasil you researched more about then i will let to you merge the PR in case you want do reverse it back.

Reply:
60 minutes is not bad, so I'll let it as it is.